### DISCUSSION TITLE
logseq integration (longterm)

### DISCUSSION THREAD

**Discussion by molleweide** [nvim-neorg/neorg#489] [Ideas]
Hi guys,

I recently stumbled upon https://github.com/logseq/logseq which is similar to `Obsidian` but FOSS and integrates with `org mode`.

I had never heard of `logseq` before and so I wanted to spread the word about it here because their philosophy seems great and over time maybe, with some more research, it could be worth while to look at integrating with `logseq`. The reason why I write this is because they replied to me and said that mobile app support for both android/ios is on the table, which could mean that we'd be able to use `neorg` with family members who aren't coders in the future which to me would be quite nice, and also to many others I imagine.

**Comment by molleweide** [comment]
vim binds for logseq. plugins are written in typescript it seems.

https://github.com/search?q=logseq+vim

**Comment by Moelf** [comment]
Logseq doesn't consume `norg` files and I doubt that would change in any time soon :(
